crates/bitwarden-uniffi/src/vault/mod.rs

```diff
@@ -37,4 +39,14 @@ impl ClientVault {
     pub fn sends(self: Arc<Self>) -> Arc<sends::ClientSends> {
         Arc::new(sends::ClientSends(self.0.clone()))
     }
+
+    /// Generate a TOTP code from a provided key.
+    ///
+    /// The key can be either:
+    /// - A base32 encoded string
+    /// - OTP Auth URI
+    /// - Steam URI
+    pub async fn generate_totp(&self, key: String) -> TotpResponse {
```

Comment:
Should we also add the time here, in case any client wants to generate the previous/next code? Or maybe as a separate function?